src/dmd/parse.d

```diff
@@ -5098,7 +5098,7 @@ class Parser(AST) : Lexer
             check(TOK.goesTo);
             if (token.value == TOK.leftCurly)
             {
-                deprecation("`(args) => { ... }` is a lambda that returns a delegate, not a multi-line lambda.");
+                deprecation("Using `(args) => { ... }` is an ambiguous way to create a lambda that returns a delegate.");
```

Comment:
"is an ambiguous **way**" does not sound very professional. I would say: "Using ... ambiguously creates a lambda that returns a delegate "

Reply:
See conversation with Adam.